lib/ecto/multi.ex

```diff
@@ -602,6 +602,18 @@ defmodule Ecto.Multi do
   defp format_operation(other),
     do: other
 
+  @doc """
+  Adds a value to the changes so far under the given name.
+
+  ## Example
+
+      Ecto.Multi.put(multi, :password, "sghbj15")
+  """
+  @spec put(t, name, any) :: t
+  def put(multi, name, value) do
+    Ecto.Multi.run(multi, name, fn _, _ -> {:ok, value} end)
```

Comment:
We can probably do an optimized implementation that stored the value directly. Can you please take a look?

Reply:
Maybe we could also support keyword lists/maps?